download/upgradeRecipe/upgradeRecipe7.adoc

```diff
@@ -2982,9 +2982,21 @@ uiScoreDirectorFactory = solver.getScoreDirectorFactory();
 ----
 
 After in `*.java`:
-[source, java]
+
+[source,java]
 ----
 SolverFactory<VehicleRoutingSolution> solverFactory = SolverFactory.createFromXmlResource(...);
 uiScoreDirectorFactory = solverFactory.getScoreDirectorFactory();
 ...
 ----
+
+== From 7.37.0.Final to 7.38.0.Final
+
+[.upgrade-recipe-major]
```

Comment:
This one is minor, not major: almost no one uses DoubleScore's. Probably no one even.